app.rb

```diff
@@ -0,0 +1,155 @@
+require './book'
+require './classroom'
+require './base_decorator'
+require './person'
+require './rental'
+require './student'
+require './teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def start
+    selecting
+  end
+
+  def books_list
+    puts 'No books available!!' if @books.empty?
+    puts "There are #{@books.count} books available"
+
+    @books.each_with_index do |book, index|
+      puts "#{index + 1}) Title: \"#{book.title}\" | Author: #{book.author}"
+    end
+  end
```

Comment:
- [ ] Your app exhibits unexpected behavior. It quits after I try to list a book. According to the video shared with you, after an option has been processed, the app should still be running and present the user with the list of choices again. Kindly fix this.

  ![levy-1](https://user-images.githubusercontent.com/81468675/174326506-350f77d8-fe77-4373-a6f6-5a3443257b99.jpg)